### PR TITLE
MM-T884  Channel name validates against two user IDs being used in URL

### PR DESCRIPTION
**Pull request by openmohan**
#### Summary

Channel name validates against two user IDs being used in URL - E2E

#### Ticket Link

[Usecase](https://automation-test-cases.vercel.app/test/MM-T884)

### CODE REVIEW

File: e2e/cypress/integration/channel/channel_name_validations_spec.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        // # Add the new channel name with invalid name and press Create Channel
+        cy.get('.new-channel__modal #newChannelName').should('be.visible').type('uzsfmtmniifsjgesce4u7yznyh__uzsfmtmniifsjgesce5u7yznyh', {force: true}).wait(TIMEOUTS.HALF_SEC);
+        cy.get('.new-channel__modal #submitNewChannel').should('be.visible').click();
+
+        // # Assert the error occured with the appropriate message
+        cy.get('.has-error').scrollIntoView().should('be.visible').within(() => {
+            cy.get('#createChannelError').should('have.text', 'Invalid channel name. User ids are not permitted in channel name for non-direct message channels.');
+
+            // # close the create channel modal
+            cy.get('#cancelNewChannel').click();
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/channel_name_validations_spec.js:70]: This is failing. Should this be outside `.has-error`. Also, would be better to use `cy.findByText('Cancel').click();`

File: e2e/cypress/integration/channel/channel_name_validations_spec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        });
+    });
+});
```
Comment (saturninoabril) [inline e2e/cypress/integration/channel/channel_name_validations_spec.js:73]: Line ending needed here.

File: e2e/cypress/integration/channel/channel_name_validations_spec.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (openmohan) [reply e2e/cypress/integration/channel/channel_name_validations_spec.js]: You mean this file to folder `channel_settings`? It was not there, so I created it.

**Comment by openmohan** [comment]
> Thanks @openmohan! Looking good, except for few comments.

Thanks for the review @saturninoabril

**Pull request merged**